### PR TITLE
add mlops lecture livecodes

### PR DESCRIPTION
**Pull request by gmanchon**
move from https://github.com/lewagon/data-lecture-starters/pull/30

### CODE REVIEW

File: .challengify_iterate.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ iterate:
 
     # cloud training
     # shell:                    02-Cloud-training/01-Shell-practice
-    cloud_lecture:            02-Cloud-training/01-Lecture-livecode
+    cloud_lecture:            02-Cloud-training/Lecture-livecode
```
Comment (krokrob) [inline .challengify_iterate.yml:28]: does myriad handle challenge without numbers?

File: .challengify_iterate.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ iterate:
 
     # automate model lifecycle
     # git:                      03-Automate-model-lifecycle/01-Git-practice
-    lifecycle_lecture:        03-Automate-model-lifecycle/01-Lecture-livecode
+    lifecycle_lecture:        03-Automate-model-lifecycle/Lecture-livecode
```
Comment (krokrob) [inline .challengify_iterate.yml:39]: same here

**Review by krokrob — COMMENTED** [review]
1 question

File: .challengify_iterate.yml (hunk 1 above)
Comment (gmanchon) [reply .challengify_iterate.yml:28]: omg, well played, I missed that ! 🙏

**Comment by gmanchon** [comment]
@krokrob I renamed the challenges in this PR + https://github.com/lewagon/data-meta/pull/249 + https://github.com/lewagon/data-solutions/pull/1110

File: notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb [hunk 3]
```diff
@@ -0,0 +1,498 @@
+{
```
Comment (krokrob) [inline notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <p>no more <code>model</code> directory </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (krokrob) [inline notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <p>replace interace w/ interface</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (krokrob) [inline notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    from taxifare_model.interface.main import (preprocess, train, evaluate)
</pre><p>from taxifare.interface</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (krokrob) [inline notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <p>same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (krokrob) [inline notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <p>sane here 😂</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (krokrob) [inline notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    from taxifare_model.interface.main import (preprocess, train, evaluate)
</pre><p>same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

File: notebooks/Lecture-livecode-Cloud-Training.ipynb [hunk 4]
```diff
@@ -0,0 +1,430 @@
+{
```
Comment (krokrob) [inline notebooks/Lecture-livecode-Cloud-Training.ipynb:1]: <p>/model does not exist anymore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (krokrob) [inline notebooks/Lecture-livecode-Cloud-Training.ipynb:1]: <p>I can't see <code>model_target</code> directory neither </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (krokrob) [inline notebooks/Lecture-livecode-Cloud-Training.ipynb:1]: <p>same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (krokrob) [inline notebooks/Lecture-livecode-Cloud-Training.ipynb:1]: <p>same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewagon/taxi-fare/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Comment by krokrob** [comment]
@gmanchon review done

File: notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb (hunk 3 above)
Comment (gmanchon) [reply notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <p>thx :pray:</p>

Comment (gmanchon) [reply notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <p>😅</p>

Comment (gmanchon) [reply notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:1]: <p>🙏</p>

File: notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb [hunk 5]
```diff
@@ -18,13 +18,13 @@
...
     "``` bash\n",
-    "cd mlops-lifecycle-lecture/model\n",
+    "cd mlops-lifecycle-lecture\n",
     "```\n",
     "\n",
     "Legacy batches:\n",
     "\n",
     "``` bash\n",
-    "cd data-challenges/07-ML-Ops/03-Automate-model-lifecycle/01-Lecture-livecode/model\n",
+    "cd data-challenges/07-ML-Ops/03-Automate-model-lifecycle/01-Lecture-livecode\n",
```
Comment (krokrob) [inline notebooks/Lecture-livecode-Automate-model-lifecycle.ipynb:27]: Doesn't `cd <local_path_to('07-ML-Ops/03-Automate-model-lifecycle/01-Lecture-livecode')>` do the trick?

File: notebooks/Lecture-livecode-Cloud-Training.ipynb [hunk 6]
```diff
@@ -18,13 +18,13 @@
...
     "``` bash\n",
-    "cd mlops-cloud-lecture/model\n",
+    "cd mlops-cloud-lecture\n",
     "```\n",
     "\n",
     "Legacy batches:\n",
     "\n",
     "``` bash\n",
-    "cd data-challenges/07-ML-Ops/02-Cloud-training/01-Lecture-livecode/model\n",
+    "cd data-challenges/07-ML-Ops/02-Cloud-training/01-Lecture-livecode\n",
```
Comment (krokrob) [inline notebooks/Lecture-livecode-Cloud-Training.ipynb:27]: same here

**Review by krokrob — CHANGES_REQUESTED** [review]
@gmanchon thanks, I have a question and you have a conflict

**Comment by gmanchon** [comment]
@krokrob good to go, I had to locally merge mlops-reorg into the branch for the conflict to appear 🤔

**Pull request merged**